src/core/utils/EventManager.ts

```diff
@@ -82,12 +82,18 @@ export class EventManager {
     _matchIntent(customEvent: CustomEvent): Intent {
         // TODO: this value is an implicit any!
         let payload = customEvent.detail;
-        let name;
+        let name = customEvent.type;
```

Comment:
Then what is the point of https://github.com/odoo-dev/jabberwock/pull/34/commits/ae601e0948b7beeb34b3ef726552c7cefdea6bcf#diff-cb93f8d40d99945285b71140ed2bb049R133 ?

Reply:
Sorry, that's a rebase mistake.

Reply:
👍